functions/src/export-csv.ts

```diff
@@ -83,7 +86,7 @@ export async function exportCsvHandler(
   // memory than iterating over and streaming both LOI and submission`
   // collections simultaneously, but it's easier to read and maintain. This will
   // likely need to be optimized to scale to larger datasets.
-  const submissionsByLocationOfInterest: {[name: string]: any[]} = {};
+  const submissionsByLocationOfInterest: { [name: string]: any[] } = {};
```

Comment:
I think gts style doesn't include these spaces.. can you run `npm run lint:fix` in the `functions` folder to check?

Reply:
Done!